### ISSUE TITLE
[Bug]: Krypton Form - ButtonSpec (And Icon), are Cut Off when maximised

### ISSUE THREAD

**Issue by MattH-Work** [Krypton-Suite/Standard-Toolkit#333]
**Describe the bug**
When maximising a Krypton Form, ButtonSpecs & icons are cut off (see image)
I have marked in Red the space between the form icon (left) and buttonspec text / icon (right) relative to the edge of the form both 'normal' & maximised, top view  is normal, bottom view is maximised

![Cropped](https://user-images.githubusercontent.com/84494183/132703351-fb13d69c-e9de-4584-b005-2c3130f89016.png)

**Comment by Smurf-IV** [comment]
@MattH-Work 
- What version have you found this in?
- What OS
- Which TFM
- And is the App DPI / High DPI / etc aware via Manifest or TFM settings ?
- And Which Theme

Cheers

**Comment by MattH-Work** [comment]
Krypton toolkit Canary – 6.0.2109
Windows 10
Not sure what TFM is (hang head in shame)
Out of the box, DPI aware, not used on High DPI monitors
Out of the box theme

Cheers

**Comment by sdreb3421** [comment]
I've also found this to be true for all versions.

**Comment by Smurf-IV** [comment]
FYI: TFM https://docs.microsoft.com/en-us/nuget/reference/target-frameworks

I have found that different themes deal with the icon in different ways.
The ButtoneSpec is always cut of if using a full height icon
The Form max / mins are also cut off in different ways dependent on the the theme.

**Comment by sdreb3421** [comment]
@Smurf-IV @Wagnerp it looks like it's an overall form issue, not just a button spec icon issue. Looks like about 10px cut off around all edges when maximized. Do you guys see that? If you guys can't see it at all I'll go investigate what OS/TFM this applies to.

**Comment by Smurf-IV** [comment]
> Do you guys see that?

Yes - Currently investigating.

**Comment by Smurf-IV** [comment]
Fixed: Notice that 
- the Close button now shows Curvature
- The icon for the FullHeight ButtonSpec is visible
![image](https://user-images.githubusercontent.com/2418812/132990902-dab8adfb-bba0-41dc-9a16-dff472c49839.png)
- The Form icon is fully visible
![image](https://user-images.githubusercontent.com/2418812/132990926-55883140-6639-4e7a-a516-644e3146147a.png)

**Comment by sdreb3421** [comment]
@Smurf-IV is this included in the nightly build? I'm not seeing a difference on my end for Nightly 266. The picture on the left is form not maximized, the picture on the right is the form maximized. I investigated the behavior of a regular form and it also seems to shrink the header, but it does not shrink the left/right/bottom edges.

![Form border](https://user-images.githubusercontent.com/1911567/134781877-1ae6eae2-b8a5-40f2-a0a7-3d7e0f8117b0.png)
.

**Comment by Wagnerp** [comment]
@Smurf-IV Is this a padding bug?

**Comment by Smurf-IV** [comment]
> @Smurf-IV Is this a padding bug?

No, It was theme related, I thought I managed to get it int the base, but if this is a "Special theme" then perhaps "the fix" needs to be moved into the general area.
Please check the changes in #337 to make sure they got into the nightly.

**Comment by sdreb3421** [comment]
Not sure if my original statement was totally clear. It's the entire form's border edges, not just the header that is being cut off. Thanks

**Comment by Wagnerp** [comment]
> > @Smurf-IV Is this a padding bug?
> 
> No, It was theme related, I thought I managed to get it int the base, but if this is a "Special theme" then perhaps "the fix" needs to be moved into the general area.
> Please check the changes in #337 to make sure they got into the nightly.

They were pushed into nightly, maybe a custom theme?

**Comment by Smurf-IV** [comment]
Just tested with the themes that come with "6.2109.268-alpha"
And it appears the fix is working as expected on Win 10 x64 1809

@sdreb3421 some Questions:
1) What OS are you using, 
2) What Resolution of monitor are you using, 
3) Is you app manifest been set to dpi aware (via manifest or code if .net 5), 
4) Are the Forms been set to dpi aware (Not font) scaling
